DIRECTORY_TREE.md

```diff
@@ -0,0 +1,1705 @@
+│  │  │  │  ├─ market_pooling
+│  │  │  │  │  ├─ market_pool_liquidity.tsx
+│  │  │  │  │  ├─ scalar_market_pool_liquidity.tsx
+│  │  │  │  │  └─ user_pool_data.tsx
+│  │  │  │  ├─ market_profile
+│  │  │  │  │  ├─ market_status
+│  │  │  │  │  │  ├─ closed.tsx
+│  │  │  │  │  │  └─ open.tsx
+│  │  │  │  │  ├─ market_view.tsx
+│  │  │  │  │  └─ market_view_container.tsx
+│  │  │  │  ├─ market_sell
+│  │  │  │  │  ├─ market_sell.tsx
+│  │  │  │  │  ├─ position_selection_box
+│  │  │  │  │  │  └─ index.tsx
+│  │  │  │  │  └─ scalar_market_sell.tsx
+│  │  │  │  └─ market_verify
+│  │  │  │     ├─ market_verify.tsx
+│  │  │  │     ├─ market_verify_container.tsx
+│  │  │  │     └─ option
+│  │  │  │        ├─ dxdao_curation.tsx
+│  │  │  │        └─ kleros_curation.tsx
+│  │  │  ├─ modal
+│  │  │  │  ├─ common_styled
+│  │  │  │  │  └─ index.tsx
+│  │  │  │  ├─ index.ts
+│  │  │  │  ├─ modal_connect_wallet
+│  │  │  │  │  ├─ img
+│  │  │  │  │  │  ├─ authereum.svg
+│  │  │  │  │  │  ├─ metamask.svg
+│  │  │  │  │  │  └─ wallet_connect.svg
+│  │  │  │  │  └─ index.tsx
+│  │  │  │  ├─ modal_deposit_withdraw
+│  │  │  │  │  └─ index.tsx
+│  │  │  │  ├─ modal_title
+│  │  │  │  │  └─ index.tsx
+│  │  │  │  ├─ modal_transaction
+│  │  │  │  │  └─ index.tsx
+│  │  │  │  ├─ modal_wrapper
+│  │  │  │  │  └─ index.tsx
+│  │  │  │  └─ modal_your_connection
+│  │  │  │     └─ index.tsx
+│  │  │  └─ settings
+│  │  │     └─ settings_view
+│  │  │        └─ index.tsx
+│  │  ├─ contexts
+│  │  │  └─ Apollo.tsx
+│  │  ├─ global.d.ts
+│  │  ├─ hooks
+│  │  │  ├─ connectedWeb3.tsx
+│  │  │  ├─ index.ts
+│  │  │  ├─ use24hsVolume.ts
+│  │  │  ├─ useAsyncDerivedValue.tsx
+│  │  │  ├─ useBalance.tsx
+│  │  │  ├─ useBlockchainMarketMakerData.tsx
+│  │  │  ├─ useCheckContractExists.tsx
+│  │  │  ├─ useCollateral.tsx
+│  │  │  ├─ useCollateralBalance.tsx
+│  │  │  ├─ useContracts.tsx
+│  │  │  ├─ useCpk.tsx
+│  │  │  ├─ useCpkAllowance.tsx
+│  │  │  ├─ useCpkProxy.tsx
+│  │  │  ├─ useDetectAdblocker.tsx
+│  │  │  ├─ useDocumentDescription.tsx
+│  │  │  ├─ useDocumentTitle.tsx
+│  │  │  ├─ useFundingBalance.tsx
+│  │  │  ├─ useGraphFpmmTransactionsFromQuestion.tsx
+│  │  │  ├─ useGraphMarketMakerData.tsx
+│  │  │  ├─ useGraphMarketUserTxData.tsx
+│  │  │  ├─ useGraphMarketsFromQuestion.tsx
+│  │  │  ├─ useGraphMeta.tsx
+│  │  │  ├─ useKlerosCuration.tsx
+│  │  │  ├─ useMarketMakerData.ts
+│  │  │  ├─ useMarkets.tsx
+│  │  │  ├─ useMultipleQueries.tsx
+│  │  │  ├─ useOutsideAlerter.tsx
+│  │  │  ├─ usePolling.ts
+│  │  │  ├─ usePolling.tsx
+│  │  │  ├─ useRealityLink.tsx
+│  │  │  ├─ useSafeApp.tsx
+│  │  │  ├─ useSymbol.tsx
+│  │  │  ├─ useTheme.tsx
+│  │  │  ├─ useTokens.tsx
+│  │  │  └─ useWindowDimensions.tsx
+│  │  ├─ index.tsx
+│  │  ├─ pages
+│  │  │  ├─ index.ts
+│  │  │  ├─ market_details_page.tsx
+│  │  │  └─ market_sections
+│  │  │     ├─ market_buy_container.tsx
+│  │  │     ├─ market_pool_liquidity_container.tsx
+│  │  │     └─ market_sell_container.tsx
```

Comment:
I think we should also include the other `market_..._container` components here as well, e.g. `market_bond_container`